Dockerfile

```diff
@@ -14,11 +14,27 @@
 
 FROM golang:1.13.4-stretch as builder
 WORKDIR /go/src/github.com/kubernetes-sigs/aws-efs-csi-driver
+
+# Cache go modules
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+
```

Comment:
So that we don't have to download go modules every time building the application.

Reply:
More details of this trick can be found in this article. 
https://medium.com/@petomalina/using-go-mod-download-to-speed-up-golang-docker-builds-707591336888

> Go 1.11 introduces the go mod downloadcommand, which takesgo.modand go.sumfiles and downloads the dependencies from them instead of using the source code. As these files don’t change frequently (unless you are updating the dependencies), they can be simply cached by the COPYcommand from Dockerfile.

```
COPY go.mod . # <- COPY go.mod and go.sum files to the workspace
COPY go.sum .

# Get dependancies - will also be cached if we won't change mod/sum
RUN go mod download
# COPY the source code as the last step
COPY . .
```